source/CustomLink.h

```diff
@@ -0,0 +1,75 @@
+/* CustomLink.h
+Copyright (c) 2014 by Michael Zahniser
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
```

Comment:
```suggestion
/* CustomLink.h
Copyright (c) 2022 by Michael Zahniser

Endless Sky is free software: you can redistribute it and/or modify it under the
terms of the GNU General Public License as published by the Free Software
Foundation, either version 3 of the License, or (at your option) any later version.

Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.  See the GNU General Public License for more details.
*/
```
I believe you can put your own name in the copyright header, but you can keep it as-is if you want to.

Reply:
My general perspective is that if you made the file, you can (and perhaps even should) put your name in the header; but you don't have to.

Reply:
I find it really weird to copyright for smbd else, maybe we should not write a name at all if copyright is denied?